pkgs/applications/virtualization/colima/default.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib
+  , buildGoModule
+  , fetchFromGitHub
+  , installShellFiles
+  , go
+  , lima
+  , makeWrapper
+}:
```

Comment:
formatting is off

Reply:
How to format it correctly? I now used `nixpkgs-fmt pkgs/applications/virtualization/colima`. Better?

Reply:
yes, lots! 
Thanks